include/ignition/gui/qml/PlottingInterface.qml

```diff
@@ -0,0 +1,539 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+import QtQuick 2.9
+import QtCharts 2.2
+import QtQuick.Controls 2.2
+import QtQuick.Controls.Styles 1.4
+import QtQuick.Controls.Material 2.1
+import QtQuick.Layouts 1.3
+import Qt.labs.platform 1.1
```

Comment:
```suggestion
import Qt.labs.platform 1.0
```

Ubuntu Bionic has version 1.0.

I also had to install an extra dependency: `qml-module-qt-labs-platform`. Can you add it here?

https://github.com/ignitionrobotics/ign-gui/blob/58cedf4c96b4baf46a0c5c79234faf9e3dc4e924/.github/ci/packages.apt#L12-L22

Reply:
ok done